core/domain/subscription_services.py

```diff
@@ -206,6 +236,10 @@ def record_user_has_seen_notifications(user_id, last_seen_msecs):
     """Updates the last_checked time for this user (which represents the time
     the user last saw the notifications in the dashboard page or the
     notifications dropdown).
+    Args:
+        user_id: str. User IDself.
+        last_seen_msecs: str. Last seen of the user when he saw the
+        notifications.
```

Comment:
This description needs updating to be more precise (it's not clear what "last seen of the user" means). Also, please indent the continuation line here by 4 (this only applies to individual arg descriptions and exceptions).

In general, please look at docstrings elsewhere in the codebase and follow the format exactly.

Reply:
Done!